src/hooks/constants/lib/fetchConstants.ts

```diff
@@ -0,0 +1,20 @@
+import { ApiPromise } from '@polkadot/api';
+import { Constants, Fee } from '../../../generated/graphql';
+
+export const repayFeeDataType = '(u32, u32)';
+
+export const unwrapRepayFeeToFee = (apiInstance: ApiPromise): Fee => {
+  const apiRepayFee = apiInstance
+    .createType(repayFeeDataType, apiInstance.consts.lbp.getRepayFee)
+    .toHuman() as Array<string>;
+
+  return {
+    numerator: apiRepayFee[0],
+    denominator: apiRepayFee[1],
+  };
+};
+export const fetchConstants = (apiInstance: ApiPromise): Constants => ({
```

Comment:
why is this called `fetchConstants` if we call most of our functions as `get...`? please rename it

Reply:
analysis:  some files do use `fetch...`,  I imitated those.

example:
https://github.com/galacticcouncil/Basilisk-ui/blob/develop/src/hooks/balances/lib/getBalancesByAddress.tsx

solution:
- change this `fetchConstants` `getConstants`
- shall I also change other occurrences of `fetch...` elsewhere in the codebase as part of this PR ?

Reply:
We're rewriting the data layer piece by piece, extension & balances are the latest examples of code separation & naming.